app/assets/javascripts/backbone/views/calendar_view.js.coffee

```diff
@@ -6,6 +6,7 @@ class MS.Views.CalendarView extends Backbone.View
     # Initialize calendar
     @$calendar = @$('#calendar')
     @stepModal = params.stepModal
+    @urlComponent = params.urlComponent
```

Comment:
@hooverlunch: Throughout Loans and BasicProjects, we will need to specify `admin/loans/some-path` and `admin/basic-projects/some-path`. Do you have a preferred method?

The method used to pass a string to the JavaScript to define the desired project-specific portion of the path.

An alternative:
```
if (params.projectType == 'BasicProject')
  @urlComponent = 'basic-projects'
else
  @urlComponent = 'loans'
```

Still another alternative is to have an admin/projects path that can do the necessary work to decide which type of project it is and forward to the appropriate controller. This method feels the cleanest.